azurerm/internal/services/redis/redis_cache_resource.go

```diff
@@ -100,6 +100,11 @@ func resourceRedisCache() *schema.Resource {
 				}, false),
 			},
 
+			"replicas_per_master": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
```

Comment:
what is the allowable range here? i presume >= 1, but is there a maximum? could we validate this?

Reply:
Unfortunately there is no limit, other than what appears to be int32 in the Go SDK. The Resource Manager [template](https://docs.microsoft.com/en-us/azure/templates/microsoft.cache/2020-06-01/redis?tabs=json#rediscreateproperties-object) does not list a limit for this either.

The portal stops me from making more than 3 replicas, but I don't know where the limit comes from.

Reply:
if the portal stops at three lets do the same, could we add a validation function that insure the value is 1-3 (unless 0 is valid then 0-3)?